package.json

```diff
@@ -135,7 +135,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint"
+      "pre-commit": ""
```

Comment:
Please ddon't change this in the PR.
It may be a good idea, but it has nothing to do with the wiki pages, so it should go to another PR

Reply:
That's why I excluded  the previous file.  Could not work out how to make got commit with an skip on this